<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>Client</class>
	<widget class="QMainWindow" name="Client">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>640</width>
				<height>480</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Client</string>
		</property>
		<widget class="QWidget" name="centralWidget">
			<layout class="QVBoxLayout" name="verticalLayout">
				<item>
					<layout class="QFormLayout" name="formLayout">
						<item row="0" column="0">
							<widget class="QLabel" name="label_ip">
								<property name="text">
									<string>서버 IP</string>
								</property>
								<property name="styleSheet">
									<string>font-size: 18px;</string>
								</property>
							</widget>
						</item>
						<item row="0" column="1">
							<widget class="QLineEdit" name="ipEdit">
								<property name="styleSheet">
									<string>font-size: 18px;</string>
								</property>
							</widget>
						</item>
						<item row="1" column="0">
							<widget class="QLabel" name="label_port">
								<property name="text">
									<string>포트 번호</string>
								</property>
								<property name="styleSheet">
									<string>font-size: 18px;</string>
								</property>
							</widget>
						</item>
						<item row="1" column="1">
							<widget class="QLineEdit" name="portEdit">
								<property name="styleSheet">
									<string>font-size: 18px;</string>
								</property>
							</widget>
						</item>
					</layout>
				</item>
				<item>
					<widget class="QLabel" name="statusLabel">
						<property name="text">
							<string/>
						</property>
						<property name="styleSheet">
							<string>color: red; font-size: 16px;</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="connectBtn">
						<property name="text">
							<string>서버 접속</string>
						</property>
						<property name="styleSheet">
							<string>font-size: 18px; padding: 6px;</string>
						</property>
					</widget>
				</item>
				<item>
					<spacer name="verticalSpacer">
						<property name="orientation">
							<enum>Qt::Vertical</enum>
						</property>
						<property name="sizeHint" stdset="0">
							<size>
								<width>20</width>
								<height>40</height>
							</size>
						</property>
					</spacer>
				</item>
				<item>
					<layout class="QHBoxLayout" name="horizontalLayout">
						<item>
							<spacer name="horizontalSpacer">
								<property name="orientation">
									<enum>Qt::Horizontal</enum>
								</property>
								<property name="sizeHint" stdset="0">
									<size>
										<width>40</width>
										<height>20</height>
									</size>
								</property>
							</spacer>
						</item>
						<item>
							<widget class="QPushButton" name="exitBtn">
								<property name="text">
									<string>종료</string>
								</property>
								<property name="styleSheet">
									<string>font-size: 16px; padding: 8px 16px;</string>
								</property>
							</widget>
						</item>
					</layout>
				</item>
			</layout>
		</widget>
		<widget class="QStatusBar" name="statusBar"/>
	</widget>
	<layoutdefault spacing="6" margin="11"/>
	<resources/>
	<connections/>
</ui>